_pages/about-us/diversity.md

```diff
@@ -1,60 +1,75 @@
 ---
-title: Diversity At TTS
-navtitle: Diversity At TTS
+title: Diversity, Equity, and Inclusion at TTS
+navtitle: Diversity, Equity, and Inclusion at TTS
```

Comment:
Can we remove the at TTS part? None of the other working groups or guilds have it

Reply:
Yeah! I'll do that in a separate PR that I'll ping you on.